src/server.cpp

```diff
@@ -1671,17 +1671,68 @@ void Server::SendHUDSetParam(session_t peer_id, u16 param, const std::string &va
 	Send(&pkt);
 }
 
-void Server::SendSetSky(session_t peer_id, const video::SColor &bgcolor,
-		const std::string &type, const std::vector<std::string> &params,
-		bool &clouds)
+void Server::SendSetSky(session_t peer_id, const SkyboxParams &params)
 {
 	NetworkPacket pkt(TOCLIENT_SET_SKY, 0, peer_id);
-	pkt << bgcolor << type << (u16) params.size();
 
-	for (const std::string &param : params)
-		pkt << param;
+	// Handle prior clients here
+	if (m_clients.getProtocolVersion(peer_id) < 38) {
```

Comment:
this should be `< 39` and the version should be bumped in networkprotocol.h

Reply:
Network protocol bumped to 39; should min version be bumped as 5.0.0 clients connect fine?

Reply:
AFAIK, min should be left as-is to prevent breakage of backwards compat with 5.0.x clients (hopefully this PR doesn't break existing the `set_sky` in any way). Bumping the min version would mean that this PR would remain unmerged until 6.0.0 :P

Reply:
No, only `LATEST_PROTOCOL_VERSION` needs to be bumped.

Reply:
@ClobberXD it's API and packet compatible if older versions (of the Lua call, and client version)  are detected.

Reply:
Not resolved.